src/map/ai/helpers/pathfind.cpp

```diff
@@ -349,7 +349,7 @@ bool CPathFind::FindPath(const position_t& start, const position_t& end)
 
 bool CPathFind::FindRandomPath(const position_t& start, float maxRadius, uint8 maxTurns, uint16 roamFlags)
 {
-    auto m_turnLength = dsprand::GetRandomNumber(1, (int)maxTurns);
+    auto m_turnLength = dsprand::GetRandomNumber(1, (int)maxTurns + 1);
```

Comment:
Common practice in C++ is to add 1 to the range if you want [1, max]

So you can do:
`dsp.GetRandomNumber(maxTurns) + 1`

Reply:
updated PR.